.gitignore

```diff
@@ -114,4 +114,9 @@ env/
 
 ### Configuration files ###
 .env
-trello_config.py
+
+### VS Code ###
+.vscode
+
+### geckodriver ###
+geckodriver.log
```

Comment:
I notice you haven't git-ignored or checked in the geckodriver binary - that might be intentional if you want developers to store it somewhere on their path outside this project, but should be mentioned in the readme